soccer/gameplay/evaluation/ball.py

```diff
@@ -90,6 +90,8 @@ def angle_btw_three_pts(a, b, vertex):
         VA = math.sqrt((vertex.x - a.x)**2 + (vertex.y - a.y)**2)
         VB = math.sqrt((vertex.x - b.x)**2 + (vertex.y - b.y)**2)
         AB = math.sqrt((a.x - b.x)**2 + (a.y - b.y)**2)
+        if (VA * VA + VB * VB - AB * AB) / (2 * VA * VB) > 1:
```

Comment:
Yes this can be zero, but I'm not entirely sure when. I got:

```
ERROR:root:Play 'AdaptiveFormation' encountered an exception: <class 'ValueError'>.  Dropping and temp. blacklisting current play...
Traceback (most recent call last):
  File "/home/jay/Code/robocup-software/soccer/gameplay/composite_behavior.py", line 85, in spin
    bhvr.spin()
  File "/home/jay/Code/robocup-software/soccer/gameplay/composite_behavior.py", line 88, in spin
    self.handle_subbehavior_exception(name, exc)
  File "/home/jay/Code/robocup-software/soccer/gameplay/composite_behavior.py", line 85, in spin
    bhvr.spin()
  File "/home/jay/Code/robocup-software/soccer/gameplay/composite_behavior.py", line 67, in spin
    super().spin()
  File "/home/jay/Code/robocup-software/soccer/gameplay/fsm.py", line 66, in spin
    if transition['condition']():
  File "/home/jay/Code/robocup-software/soccer/gameplay/tactics/defensive_forward.py", line 64, in <lambda>
    evaluation.ball.robot_has_ball(self.collector.robot),
  File "/home/jay/Code/robocup-software/soccer/gameplay/evaluation/ball.py", line 97, in robot_has_ball
    math.cos(angle), math.sin(angle)), main.ball().pos, robot.pos)
  File "/home/jay/Code/robocup-software/soccer/gameplay/evaluation/ball.py", line 93, in angle_btw_three_pts
    return math.acos((VA * VA + VB * VB - AB * AB) / (2 * VA * VB))
ValueError: math domain error
```

while testing (not your change, but below). Since you're touching it, could you take a look?